libs/common/auth/auth.go

```diff
@@ -206,6 +210,16 @@ func GetAuthClaims(ctx context.Context) (*IDTokenClaims, error) {
 	}
 }
 
+// SessionValidUntil returns time.Time when the session gets marked as expired
+func SessionValidUntil(ctx context.Context) (time.Time, error) {
+	res, ok := ctx.Value(TokenExpires{}).(*time.Time)
+	if !ok {
+		return time.Now(), status.Error(codes.Internal, "tokenExpires not in context, set up auth")
```

Comment:
this is not a valid expected state, thus warrants a panic, not an error

Reply:
https://github.com/helpwave/services/issues/811#issuecomment-2427021965